.drone.yml

```diff
@@ -199,17 +343,39 @@ steps:
     - refs/tags/*
     event:
     - tag
-
+volumes:
+  - name: docker
+    host:
+      path: \\\\.\\pipe\\docker_engine
+depends_on:
+- build
+---
+kind: pipeline
+name: windows-amd64-2004
```

Comment:
need to use my clone: false hack to make 2004/20h2 work, looks like this

https://github.com/rancher/rancher/blob/master/.drone.yml#L738-L744

Reply:
done